docs/fitting.rst

```diff
@@ -635,47 +632,31 @@ convenience functions to perform exactly this task.  An example is shown below.
     :align: center
     :context: close-figs
 
-    import numpy as np
-    import matplotlib.pyplot as plt
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>> from astropy.modeling import models
+    >>> from astropy import units as u
+    >>> from specutils.spectra import Spectrum1D, SpectralRegion
+    >>> from specutils.fitting import fit_generic_continuum
 
-    from astropy.modeling import models
-    from astropy import units as u
+    >>> np.random.seed(0)
+    >>> x = np.linspace(0., 10., 200)
+    >>> y = 3 * np.exp(-0.5 * (x - 6.3)**2 / 0.1**2)
+    >>> y += np.random.normal(0., 0.2, x.shape)
 
-    from specutils.spectra import Spectrum1D, SpectralRegion
-    from specutils.fitting import fit_generic_continuum
+    >>> y_continuum = 3.2 * np.exp(-0.5 * (x - 5.6)**2 / 4.8**2)
+    >>> y += y_continuum
 
-    np.random.seed(0)
-    x = np.linspace(0., 10., 200)
-    y = 3 * np.exp(-0.5 * (x - 6.3)**2 / 0.1**2)
-    y += np.random.normal(0., 0.2, x.shape)
+    >>> spectrum = Spectrum1D(flux=y*u.Jy, spectral_axis=x*u.um)
 
-    y_continuum = 3.2 * np.exp(-0.5 * (x - 5.6)**2 / 4.8**2)
-    y += y_continuum
+    >>> g1_fit = fit_generic_continuum(spectrum)
 
-    spectrum = Spectrum1D(flux=y*u.Jy, spectral_axis=x*u.um)
+    >>> y_continuum_fitted = g1_fit(x*u.um)
 
-    g1_fit = fit_generic_continuum(spectrum)
-
-    y_continuum_fitted = g1_fit(x*u.um)
-
-    plt.plot(x, y)
-    plt.plot(x, y_continuum_fitted)
-    plt.title('Continuum Fitting')
-    plt.grid(True)
-
-The normalized spectrum is simply the old spectrum devided by the
-fitted continuum, which returns a new object:
-
-.. plot::
```

Comment:
Why was this removed?  I think this is here specifically to demonstrate that you can do continuum normalization.

Reply:
Added back -- not sure how I missed it.